python/test/Ice/packagemd/Package.ice

```diff
@@ -2,6 +2,8 @@
 
```

Comment:
The `packagemd` test is specifically about testing `python:package`, so I left it alone,
except for suppressing the deprecation warnings (I checked, and without the `suppress`, the deprecation warning was being correctly emitted).